### PR TITLE
Draft of 2021-03 snapshot release

### PR DESCRIPTION
**Pull request by dstebila**

### CODE REVIEW

**Comment by dstebila** [comment]
@xvzcf, @baentsch, @christianpaquin Once you all approve this PR, I'll merge and tag the release.

**Comment by dstebila** [comment]
Note to self: don't forget to update release date in release notes

File: RELEASE.md [hunk 1]
```diff
@@ -13,16 +13,15 @@ The **Open Quantum Safe (OQS) project** has the goal of developing and prototypi
...
+This is the fifth snapshot release of OQS-OpenSSL_1\_1\_1-stable.  It is based on OpenSSL 1.1.1j.
 
-- Uses the updated NIST Round 2 submissions added to liboqs 0.4.0, as described in the [liboqs release notes](https://github.com/open-quantum-safe/liboqs/blob/main/RELEASE.md).
-
-Deprecations
-------------
-
-As a result of NIST's announcement of Round 3 of the Post-Quantum Cryptography Standardization Project, this is the last release of OQS-OpenSSL that contain algorithms from Round 2 that are not Round 3 finalists or alternate candidates. Those algorithms will be removed in the next release. The algorithms in question are: NewHope, ThreeBears, MQDSS, and qTesla. These algorithms are considered deprecated within OQS-OpenSSL will receive no updates after this release.
+- Removes algorithms from NIST PQC Round 2 that did not advance to Round 3: NewHope, ThreeBears, MQDSS, qTesla.
+- Updates algorithms to those use in liboqs 0.5.0, as described in the [liboqs release notes](https://github.com/open-quantum-safe/liboqs/blob/main/RELEASE.md).
```
Comment (baentsch) [inline RELEASE.md]: typo: "use" -> "used"

**Review by baentsch — DISMISSED** [review]
As you asked for comment, I find it inconvenient (unusual?) to only have the release notes for this (latest) release in this file and not all release notes (dated, under subtitles by release, i.e., an ever growing list) in the same file (see [e.g. here](https://github.com/openssl/openssl/blob/master/CHANGES.md). This way, significant changes could be added by PR authors/reviewers to the file and a release would only necessitate adding a new heading. Readers in turn would be able to see progress over time in one place. But as a) all main changes since the last release have been captured now and b) it is done like this also in liboqs I don't want this to hold up the release.

**Comment by dstebila** [comment]
I've added links to the past release notes

**Comment by dstebila** [comment]
Rebased after OpenSSL 1.1.1k merge in #298. Any final comments? If not, I'll tag and release on Friday.

**Review by baentsch — APPROVED** [review]
LGTM. `oqs-scripts/release-test.sh` also successfully executed.

**Pull request merged**